src/test/java/uk/gov/companieshouse/insolvency/delta/mapper/PractitionerAddressMapperTest.java

```diff
@@ -0,0 +1,85 @@
+package uk.gov.companieshouse.insolvency.delta.mapper;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.companieshouse.api.delta.PractitionerAddress;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = { PractitionerAddressMapperImpl.class })
+class PractitionerAddressMapperTest {
+
+    @Autowired
+    PractitionerAddressMapper mapper;
+
+    @Test
+    void shouldMapFullPractitionerAddress() {
+        PractitionerAddress sourcePractitionerAddress = new PractitionerAddress();
+        sourcePractitionerAddress.setAddressLine1("Yerrill Murphy Edelman House");
+        sourcePractitionerAddress.setAddressLine2("1238 High Road");
+        sourcePractitionerAddress.setLocality("Whetstone");
+        sourcePractitionerAddress.setRegion("London");
```

Comment:
I know we can't use Lombok to use builders etc, but suggest to create separate TestData classes like PractitionerAddressTestData class which has static methods to give different objects. and in test assertions we check target.getXXX equals to source.getYYY, that all test data like "London", "Whetstone" be in TestData class for each model(s)